test/e2e_node/jenkins/cri_validation/benchmark-config.yaml

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
Yeah, @coufon found that the benchmark result is not quite stable if we run them one after another. If we run them on the same VM, the result will be quite fluctuated.